.github/workflows/schemacode_ci.yml

```diff
@@ -99,6 +88,34 @@ jobs:
           path: coverage.xml
         if: success()
 
+  publish:
+    runs-on: ${{ matrix.os }}
+    name: Publish Python Package
+    needs: [test]
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.10"]
+    steps:
+      - name: "Fetch packages"
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      - name: "Test PyPI upload"
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          skip_existing: true
+      - name: "Upload to PyPI (on tags)"
+        if: ${{ startsWith(github.ref, 'refs/tags/schema-') }}
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
I'm avoiding pushing to proper PyPI by not having this secret in my local repository. If you merge and test tag, you need to remove yours.

Reply:
Or I just need it to have a different name, right?

Reply:
Sure.

Reply:
Yeah, it has a different name :3